doc/examples.xml

```diff
@@ -313,6 +313,51 @@ gap> GeneralisedPetersenGraph(IsMutableDigraph, 9, 4);
 </ManSection>
 <#/GAPDoc>
 
+<#GAPDoc Label="BishopsGraph">
+<ManSection>
+  <Oper Name="BishopsGraph" Arg="[filt, ]color, m, n"/>
+  <Returns>A digraph.</Returns>
+  <Description>
+    If <A>m</A> and <A>n</A> are positive integers and <A>color</A> is
+    either the string <C>"dark-square"</C> or the string <C>"light-square"</C>,
+    then this operation returns the <E>bishop's graph</E> for an <A>m</A> by
+    <A>n</A> board for a bishop of that square color.<P/>
+
+    From
+    <URL>https://mathworld.wolfram.com/BishopGraph.html</URL>:
+    <P/>
+
+    <Q> A bishop's graph is a graph formed from possible moves of a bishop chess
+    piece, which may make diagonal moves of any length on a chessboard (or any 
+    other board). To form the graph, each chessboard square is considered a vertex, 
+    and vertices connected by allowable bishop moves are considered edges. Because 
```

Comment:
```suggestion
    and there is an edge (or path?) between two vertices if and only if one can be reached by the other by an allowable move of a bishop. Because 
```